### ISSUE TITLE
Timeout for larger scan

### ISSUE THREAD

**Issue by haeser** [42Crunch/api-security-audit-action#13]
Hi, I was trying to run a scan on a larger (combined) specification file but I constantly get into the following error.
Is there a way for me to configure the amount of retries for the report?

Thank you in advance. I redacted some parts in the log

```
Ignoring network errors
Reading config file '42c-conf.yaml' in '/github/workspace'
Matched branch name
GET https://platform.us.42crunch.cloud/api/v2/sqgs/audit 200
GET https://platform.us.42crunch.cloud/api/v1/organizations/me/settings/apiNamingConvention 200
GET https://platform.us.42crunch.cloud/api/v1/organizations/me/settings/collectionNamingConvention 200
Looking for OpenAPI files in: 
Looking for OpenAPI contents in:
Discovered OpenAPI files:
Looking for the technical collection name:
POST https://platform.us.42crunch.cloud/api/v1/collections/technicalName 200
GET https://platform.us.42crunch.cloud/api/v1/collections/1dd4d329-b812-488f-86e9-3819088621ab/apis 200
GET https://platform.us.42crunch.cloud/api/v1/apis/76c3e932-ac4f-430c-9be4-6a2c1164c074 200
PUT https://platform.us.42crunch.cloud/api/v1/apis/76c3e932-ac4f-430c-9be4-6a2c1164c074 200
Reading assessment report for API ID: 76c3e932-ac4f-430c-9be4-6a2c1164c074
GET https://platform.us.42crunch.cloud/api/v1/apis/76c3e932-ac4f-430c-9be4-6a2c1164c074 200
Assessment report for API ID: 76c3e932-ac4f-430c-9be4-6a2c1164c074 is not ready, retrying.
GET https://platform.us.42crunch.cloud/api/v1/apis/76c3e932-ac4f-430c-9be4-6a2c1164c074 200
Assessment report for API ID: 76c3e932-ac4f-430c-9be4-6a2c1164c074 is not ready, retrying.
GET https://platform.us.42crunch.cloud/api/v1/apis/76c3e932-ac4f-430c-9be4-6a2c1164c074 200
Assessment report for API ID: 76c3e932-ac4f-430c-9be4-6a2c1164c074 is not ready, retrying.
GET https://platform.us.42crunch.cloud/api/v1/apis/76c3e932-ac4f-430c-9be4-6a2c1164c074 200
Assessment report for API ID: 76c3e932-ac4f-430c-9be4-6a2c1164c074 is not ready, retrying.
GET https://platform.us.42crunch.cloud/api/v1/apis/76c3e932-ac4f-430c-9be4-6a2c1164c074 200
Assessment report for API ID: 76c3e932-ac4f-430c-9be4-6a2c1164c074 is not ready, retrying.
GET https://platform.us.42crunch.cloud/api/v1/apis/76c3e932-ac4f-430c-9be4-6a2c1164c074 200
Assessment report for API ID: 76c3e932-ac4f-430c-9be4-6a2c1164c074 is not ready, retrying.
GET https://platform.us.42crunch.cloud/api/v1/apis/76c3e932-ac4f-430c-9be4-6a2c1164c074 200
Assessment report for API ID: 76c3e932-ac4f-430c-9be4-6a2c1164c074 is not ready, retrying.
GET https://platform.us.42crunch.cloud/api/v1/apis/76c3e932-ac4f-430c-9be4-6a2c1164c074 200
Assessment report for API ID: 76c3e932-ac4f-430c-9be4-6a2c1164c074 is not ready, retrying.
GET https://platform.us.42crunch.cloud/api/v1/apis/76c3e932-ac4f-430c-9be4-6a2c1164c074 200
Assessment report for API ID: 76c3e932-ac4f-430c-9be4-6a2c1164c074 is not ready, retrying.
GET https://platform.us.42crunch.cloud/api/v1/apis/76c3e932-ac4f-430c-9be4-6a2c1164c074 200
Assessment report for API ID: 76c3e932-ac4f-430c-9be4-6a2c1164c074 is not ready, retrying.
GET https://platform.us.42crunch.cloud/api/v1/apis/76c3e932-ac4f-430c-9be4-6a2c1164c074 200
Assessment report for API ID: 76c3e932-ac4f-430c-9be4-6a2c1164c074 is not ready, retrying.
GET https://platform.us.42crunch.cloud/api/v1/apis/76c3e932-ac4f-430c-9be4-6a2c1164c074 200
Assessment report for API ID: 76c3e932-ac4f-430c-9be4-6a2c1164c074 is not ready, retrying.
Error: Error: Unexpected exception "Timed out while waiting for the assessment report for API ID: 76c3e932-ac4f-430c-9be4-6a2c1164c074" 
```

**Comment by dt42c** [comment]
Hi,
Since the question is related to the use of our Enterprise platform, we have created a support ticket and sent you an email via our Support team.

Please check your inbox, and in case you haven't received that email, please let us know.

Thanks.

**Comment by ak1394** [comment]
This issue has been fixed in v3.7.0
